src/redux/books/books.js

```diff
@@ -0,0 +1,28 @@
+const ADD_BOOK = 'bookStore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+
+const initialState = [];
+
+export const addBook = (payload) => ({
+  type: ADD_BOOK,
+  payload,
+});
+
+export const removeBook = (payload) => ({
+  type: REMOVE_BOOK,
+  payload,
+});
+
+const reducer = (state = initialState, action) => {
+  if (action.type === ADD_BOOK) {
+    const newState = [...state, action.payload];
+    return newState;
+  }
+  if (action.type === REMOVE_BOOK) {
+    const uState = state.filter((stut) => stut.id !== action.payload);
+    return uState;
+  }
+  return state;
+};
```

Comment:
- [ ] **[OPTIONAL]** Have you considered using `switch case` for redux reducer instead of the `if` statement? Using the [switch case](https://www.pluralsight.com/guides/how-to-write-redux-reducer) makes it easier and simpler to use the redux store. See suggestion below:

```suggestion
const reducer = (state = initialState, action) => {
  switch (action.type) {
        case ADD_BOOK:
            return [...state, action.payload];
        case REMOVE_BOOK:
           return state.filter((stut) => stut.id !== action.payload);
        default:
            return state;
};
```

Reply:
Hi @julie-ify
Thank you for reviewing my project and for the amazing feedback. I will implement the changes as requested.